priv/package.json

```diff
@@ -36,7 +36,14 @@
     "mocha": "3.2.0",
     "style-loader": "0.13.0",
     "url-loader": "0.5.7",
-    "webpack": "1.12.9"
+    "webpack": "1.14.0",
+    "react-bootstrap": "0.30.7",
```

Comment:
You're right it is all about clarity - `webpack` and other tools are just `development` tools needed, but semantically it make sense only in *Node.js* environment.